config/routes.rb

```diff
@@ -65,7 +65,7 @@
 
     resources :statuses, only: [:show] do
       member do
-        get :activity
+        get '/activity(/:activity_id)', to: 'statuses#activity', as: :activity
```

Comment:
Couldn't/shouldn't Status updates have their own namespace instead of repurposing an existing controller method?

Reply:
Both creates and updates are activities on the status, so it makes sense to me to keep it all in one space, at least URL-wise. I mean, we united both "create" and "announce" and `/activity` anyway so it'd be weird to separate "update" now. Though I've just realized that "/activity/:id" is going to have inconsistent pluralization with the rest of the paths :(